lib/components/user/monitored-trip/trip-basics-pane.tsx

```diff
@@ -101,6 +101,19 @@ const AvailableDays = styled(FieldSet)`
     width: 100%;
   }
 `
+// Targets the formik checkboxes to provide better contrast on focus styles
+const CheckboxWrapper = styled.span`
+  input {
+    &:focus-visible,
+    &:focus {
+      outline-offset: 0.5px;
+      outline: solid 2px blue;
+      &:checked {
+        outline: solid 2px white;
+      }
+    }
+  }
+`
```

Comment:
Could you move these to the `AvailableDays` component? (it is a styled component definition) In doing so you won't need the `CheckboxWrapper`.

Reply:
Oh perfect! Thank you